sphinx/ext/autodoc/importer.py

```diff
@@ -231,12 +231,19 @@ def get_object_members(subject: Any, objpath: List[str], attrgetter: Callable,
         except AttributeError:
             pass
 
-    if analyzer:
-        # append instance attributes (cf. self.attr1) if analyzer knows
-        namespace = '.'.join(objpath)
-        for (ns, name) in analyzer.find_attr_docs():
-            if namespace == ns and name not in members:
-                members[name] = Attribute(name, True, INSTANCEATTR)
+    # append instance attributes (cf. self.attr1) if analyzer knows
+    for i, cls in enumerate(getmro(subject)):
```

Comment:
There is a possibility that `getmro()` raises `AttributeError`. try-except is needed here.

Reply:
Wait. This change is not needed for this change. This should be reverted.